### PR TITLE
added documents for enhancements

### PR DESCRIPTION
**Pull request by britt6612**
- Added `radiobutton.check.background.color` to RadioButton. ([#5243](https://github.com/grommet/grommet/pull/5243))
- Added `tag.icon.remove` to Tag to allow the icon to be provided from the theme. ([#6960](https://github.com/grommet/grommet/pull/6960))

### CODE REVIEW

File: src/screens/Tag.js [hunk 1]
```diff
@@ -173,6 +173,14 @@ const TagPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="tag.icons.remove">
+          <Description>The remove icon in the Tag.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>{`{FormClose}`}</Example>
```
Comment (taysea) [inline src/screens/Tag.js:179]:
```suggestion
            <Example defaultValue>{`FormClose`}</Example>
```

File: src/screens/Tag.js [hunk 2]
```diff
@@ -173,6 +173,14 @@ const TagPage = () => (
...
+            <Example>{`{Trash}`}</Example>
```
Comment (taysea) [inline src/screens/Tag.js:180]:
```suggestion
            <Example>{`Trash`}</Example>
```

**Review by taysea — APPROVED** [review]

**Review by jcfilben — COMMENTED** [review]
Changes look good 

We should also document changes from these PRs:
https://github.com/grommet/grommet/pull/6097
https://github.com/grommet/grommet/pull/6887
https://github.com/grommet/grommet/pull/6956
https://github.com/grommet/grommet/pull/6952

**Comment by britt6612** [comment]
Added the rest looks like `level` is already there for PageHeader @jcfilben

File: src/screens/DataTable.js [hunk 3]
```diff
@@ -92,6 +92,15 @@ const DataTablePage = () => (
           <GenericAlignSelf />
         </Property>
 
+        <Property name="allowSelectAll">
+          <Description>
+            To have the option of the select all in the DataTable header.
+          </Description>
+          <PropertyValue type="boolean">
+            <Example>allowSelectAll={false}</Example>
+          </PropertyValue>
```
Comment (jcfilben) [inline src/screens/DataTable.js:101]:
```suggestion
          <Description>
            Whether or not to show the select all CheckBox in the DataTable header when `onSelect` is defined.
          </Description>
          <GenericBoolFalse />
```

File: src/screens/Grommet.js [hunk 4]
```diff
@@ -112,6 +112,16 @@ const GrommetPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="drop">
```
Comment (jcfilben) [inline src/screens/Grommet.js:115]: This should be under the options prop.
`options.drop.checkContainingBlock`

File: src/screens/Grommet.js [hunk 5]
```diff
@@ -112,6 +112,16 @@ const GrommetPage = () => (
...
+          <Description>
+            When using `ContainerTargetContext` drop will be positioned
+            correctly next to target.{' '}
```
Comment (jcfilben) [inline src/screens/Grommet.js:118]: When enabled additional checks will be performed on the target of the Drop to more accurately determine its position. This is helpful in situations where the target is within a containing block that has a transform, perspective or filter css property applied.

 of the drop will be calculated before determining the Drop position.

File: src/screens/Grommet.js [hunk 6]
```diff
@@ -112,6 +112,16 @@ const GrommetPage = () => (
...
+          </Description>
+          <PropertyValue type="boolean">
+            <Example>"true"</Example>
+          </PropertyValue>
```
Comment (jcfilben) [inline src/screens/Grommet.js:122]:
```suggestion
          <GenericBoolFalse />
```

File: src/screens/Grommet.js [hunk 7]
```diff
@@ -264,7 +342,10 @@ const GrommetPage = () => (
       lower: "string",
       upper: "string"
     },
-    select: { multiple: "string" },
+    select: { 
+    multiple: "string",
+    selected: "string",
```
Comment (jcfilben) [inline src/screens/Grommet.js:347]:
```suggestion
      multiple: "string",
      selected: "string",
```

File: src/screens/Grommet.js (hunk 4 above)
Comment (britt6612) [reply src/screens/Grommet.js:115]: ooh gotcha

File: src/screens/Grommet.js [hunk 8]
```diff
@@ -137,19 +137,26 @@ const GrommetPage = () => (
...
+            containing block that has a transform, perspective or filter css
+            property applied.`}
           </Description>
           <PropertyValue type="object">
             <Example>
               {`
 {
   box: { cssGap: true },
-  layer: { singleId: boolean }
+  layer: { singleId: boolean ,
```
Comment (jcfilben) [inline src/screens/Grommet.js:158]:
```suggestion
  layer: { singleId: boolean },
```

**Review by jcfilben — APPROVED** [review]

**Review by jcfilben — APPROVED** [review]

**Pull request merged**